sdk/appcenter/src/main/java/com/microsoft/appcenter/http/DefaultHttpClient.java

```diff
@@ -34,6 +34,12 @@
     public static final String METHOD_DELETE = "DELETE";
 
 
+    /**
+     * retry after milliseconds duration header key.
+     */
+    public static final String RETRY_AFTER_MS = "x-ms-retry-after-ms";
+
```

Comment:
extra line

Reply:
Thanks! Just looked at the work item you mentioned above. Fixing them all now